### DISCUSSION TITLE
send_form() in 3.0.0-rc1

### DISCUSSION THREAD

**Discussion by jullanggit** [algesten/ureq#837] [Q&A] [answered]
I am currently migrating a small project of mine to the rc, but can't seem to figure out how to send form data. There is now the somewhat generic send function but it doesnt work with the &[(&str, &str)] 2.x.x required. Tldr: How do you send a form in 3.0.0-rc1?

**Comment by algesten** [comment]
Interesting. Ok. That might be an oversight. Thanks for checking, I will investigate if I missed something here.

**Reply by jullanggit** [reply]
Thank you!

**Reply by algesten** [reply, marked as answer]
This should be sorted in main

**Reply by jullanggit** [reply]
Thanks!
